Modules/+Drivers/@Wavemeter/Wavemeter.m

```diff
@@ -126,11 +126,17 @@
                 output = obj.com(cmd,obj.channel,0);
             end
             if output <= 0
-                if output == -1
-                    msg = 'Low signal';
+                if output == 0 || output == -5 || output == -6 || output == -15
```

Comment:
do we care about -7 (InfNothingChanged), -8 (ErrNoPulse), -13 (ErrDiv0)

Reply:
Feel free to add them! I covered the ones I cared most about.